src/json-patch-ot-agent.js

```diff
@@ -39,8 +39,9 @@ JSONPatchOTAgent.prototype.send = function(sequence){
 		path: this.remotePath,
 		value: this.remoteVersion
 	});
-	this.pending.push(sequence);
-	return JSONPatchQueue.prototype.send.call(this, newSequence);
+	var versionedJSONPatch = JSONPatchQueue.prototype.send.call(this, newSequence);
```

Comment:
I think it's compiler's thing to do such optimizations. It will probably figure it out (or even something better).